### PR TITLE
Incorporate cross-origin iframe support in WebOTP

### PR DESCRIPTION
**Pull request by agektmr**
Incorporate cross-origin iframe support in the WebOTP canonical article.

### CODE REVIEW

**Comment by agektmr** [comment]
cc: @yi-gu

File: src/site/content/en/blog/web-otp/index.md [hunk 1]
```diff
@@ -400,11 +385,105 @@ You may also fork it and create your version:
...
+  width="300",
+  height="600",
+  class="w-screenshot"
+%}
+  <figcaption class="w-figcaption">
+    WebOTP API within an iframe in action.
+  </figcaption>
+</figure>
+
+You can try the demo yourself at
```
Comment (mihajlija) [inline src/site/content/en/blog/web-otp/index.md:426]:
```suggestion
You can try the demo at
```

File: src/site/content/en/blog/web-otp/index.md [hunk 2]
```diff
@@ -400,11 +385,105 @@ You may also fork it and create your version:
...
+[https://web-otp-iframe-demo.stackblitz.io](https://web-otp-iframe-demo.stackblitz.io).
+
+### Annotate bound-origins to the SMS text message
+
+When WebOTP API is called from within an iframe, the SMS text message must
+include the top-frame origin preceded by `@` followed by the OTP preceded by `#`
+followed by the iframe origin preceded by `@`.
```
Comment (mihajlija) [inline src/site/content/en/blog/web-otp/index.md:433]:
```suggestion
When WebOTP API is called from within an iframe, the SMS text message must
include the top-frame origin preceded by `@` followed by the OTP preceded by `#`
and the iframe origin preceded by `@`.
```

**Review by mihajlija — APPROVED** [review]

**Comment by MeatsMountain** [comment]
src/site/content/en/blog/web-otp/index.md

**Comment by MeatsMountain** [comment]
Meats Mountain invited you to view a collection


Favorite images

By Meats Mountain · 1 item


reddit.com


I drew a QR code try to see where it leads: tommyinnit


View collection


You received this mail because Meats Mountain shared this collection with  
you. If you no longer wish to receive email notification of shared  
collections, unsubscribe here.



Get the Google Search App
Download on the App Store Get it on Google Play

**Comment by MeatsMountain** [comment]
Company

Global Network Initiative INC

**Pull request merged**